nmostesting/Config.py

```diff
@@ -383,8 +396,10 @@
     },
     "nmos-parameter-registers": {
         "repo": "nmos-parameter-registers",
-        "versions": ["main"],
-        "default_version": "main",
+        "url": "https://github.com/alabou/",
+        "branch": "bcp-004-02",
+        "versions": ["bcp-004-02"],
+        "default_version": "bcp-004-02",
```

Comment:
This should be removed once https://github.com/AMWA-TV/nmos-parameter-registers/pull/65 has been merged.

Reply:
For future reference, these parameters can be overridden in your UserConfig.py so you don't need to touch the Config.py when using development config, like this:
```
CONFIG.SPECIFICATIONS["nmos-parameter-registers"]["url"] = "https://github.com/alabou/"
CONFIG.SPECIFICATIONS["nmos-parameter-registers"]["versions"] = ["bcp-004-02"]
CONFIG.SPECIFICATIONS["nmos-parameter-registers"]["default_version"] = "bcp-004-02"
```